webpack.config.cjs

```diff
@@ -325,24 +316,14 @@ const activationNotice = {
   },
   plugins: [
     ...sharedConfig.plugins,
+    new DependencyExtractionWebpackPlugin({
```

Comment:
Should we be using the built in version of react ( wp-elements ) for activation?